servatrice/src/servatrice.cpp

```diff
@@ -405,6 +405,8 @@ bool Servatrice::initServer()
         }
     }
 #endif
+
+    qDebug() << "Idle client timeout value: " << getIdleClientTimeout();
```

Comment:
I understand that my clients idle timeout is set to 50 seconds since i read the description in the servers ini file and realize that the clients will be warned for being idle at 90% of 50 seconds?  The output is there for troubleshooting value(s) on server startup.  Same way the output for the path to the servers ini file is listed on startup, or the value for if your using a db or not.